test/java/src/test/java/ghostdriver/AuthBasicTest.java

```diff
@@ -61,4 +65,21 @@ public void simpleBasicAuthShouldWork() {
         assertTrue(driver.getPageSource().contains("authenticated"));
     }
 
+    // we should be able to interact with pages that have content security policies
+    // phantomjs loads a plain text file
+    // received "contentType":"text/plain; charset=UTF-8, text/html;charset=utf-8"
+    @Ignore
```

Comment:
twitter is returning content type `text/plain; charset=UTF-8, text/html;charset=utf-8`.  hence phantomjs loads the page as plain text instead of html.